client/src/App.test.js

```diff
@@ -6,6 +6,26 @@ import App from './App';
 const responseData = 
 {
    "clusters": [
+      {
+         "title": "Wear OS app of the week",
+         "card": {
```

Comment:
Hmm, i would've created a different card from the standard cluster card since they aren't guaranteed to be the same (they don't look the same visually, but happened to use the same fields in the current mock). But this is fine here since the backend changes were already submitted and it's outside the same of this PR.